docs/developer-setup/README.md

```diff
@@ -369,23 +370,23 @@ On Linux
 ```sh
 cmake -B build
 cmake --build build
-cmake --install build --component library --prefix $HOME/Desktop/CesiumOmniverse
+cmake --install build --component library --prefix /path/to/install/location
```

Comment:
"install location" is somewhat overloaded, can this be more specific? like `/path/to/git/repo`?

Reply:
`install` is the right term from the CMake / make / build system perspective. I don't think `/path/to/git/repo` is quite right because the installed files are usually placed somewhere else (e.g. how `genStubs.sh` is installing to `/temp`)

Reply:
oh, I think I confused myself. You're totally right